tools/setup-dev.sh

```diff
@@ -227,6 +228,14 @@ if ! ${skip_install_deps} ; then
         echo "Dependency installation failed"
         exit 1
       }
+      echo "=> make sure user in libvirt and starting libvirtd"
+      USER=`whoami`
+      sudo usermod -a -G libvirt $USER
+      newgrp libvirt
+      sudo mv /etc/libvirt/libvirtd.conf /etc/libvirt/libvirtd.conf.bak
+      sudo sed -e 's/#unix_sock_\([g|r]\)/unix_sock_\1/' /etc/libvirt/libvirtd.conf.bak > /etc/libvirt/libvirtd.conf
```

Comment:
I guess this is problematic:
```
$ sudo sed -e 's/#unix_sock_\([g|r]\)/unix_sock_\1/' /etc/libvirt/libvirtd.conf.bak > /etc/libvirt/libvirtd.conf
-bash: /etc/libvirt/libvirtd.conf: Permission denied
```
It is because you're still ubuntu user when redirecting stdout.